### DISCUSSION TITLE
Adding a feature to reset errors on the front-end only

### DISCUSSION THREAD

**Discussion by ismailnakkar** [livewire/livewire#8609] [3. Feature ideas]
Let's say a validation error occurred but the error should disappear as soon as the user starts typing something. It would helpful to hide validation for certain attributes without making calls to the backend

**Comment by lrljoe** [comment]
Perfectly possible to do with wire:dirty and wire:target already...

**Reply by ismailnakkar** [reply]
That's true, but sometimes custom function is needed and it targets errors only, so a way to handle them from the frontend would be beneficial
